CONTRIBUTING.md

```diff
@@ -91,4 +150,4 @@ More Examples:
 
 ## Fixing a commit where the DCO failed
 
-Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
+Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
```

Comment:
Do not remove the newline at the end of the file here.

```suggestion
Check out [this guide](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).

```

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline